driver/bpf/fillers.h

```diff
@@ -6785,8 +6883,26 @@ FILLER(sched_prog_fork_3, false)
 	/* Parameter 20: vpid (type: PT_PID) */
 	pid_t vpid = bpf_task_tgid_vnr(child);
 	res = bpf_val_to_ring_type(data, vpid, PT_PID);
+	if (res != PPM_SUCCESS)
+	{
+		return res;
+	}
+
+	/* Parameter 21: pid_namespace init task start_time monotonic time in ns (type: PT_UINT64) */
+	if (pidns)
+	{
+	struct task_struct *child_reaper = (struct task_struct *)_READ(pidns->child_reaper);
+		if (child_reaper)
+		{
+			pidns_init_start_time = _READ(child_reaper->start_time);
+		}
+	}
+	res = bpf_val_to_ring_type(data, pidns_init_start_time, PT_UINT64);
+	if (res != PPM_SUCCESS)
+	{
+		return res;
+	}
```

Comment:
uhm maybe I'm missing something but here it should only `return res` instead of the `if` since we need to return always something